correios/utils.py

```diff
@@ -104,3 +104,7 @@ def get_resource_path(path) -> Path:
     resource_package = "correios"
     resource_path = os.path.join("data", path)
     return Path(pkg_resources.resource_filename(resource_package, resource_path))
+
+
+def sanitize_phrase(phrase: str) -> str:
+    return re.sub("[!@#$º><%&]", "", phrase)
```

Comment:
Sei que talvez não faça muita diferença nesse caso, mas acha que seria interessante guardar a regex compilada pra ganhar uns nanosegundos de performance? =P A longo prazo teríamos economizado um tempo considerável.

```suggestion
regex_special_characters = re.compile("[!@#$º><%&]")
def sanitize_phrase(phrase: str) -> str:
    return regex_special_characters.sub("", phrase)
```

Reply:
Só declara esse RegexObject `regex_special_characters` em um lugar mais apropriado (tipo lá no topo do arquivo) e se quiser, mude o nome.

Reply:
Opa, então não sei se teria necessidade nesse caso até porque não sei se teria um ganho significativo, mas se achar que faz sentido mesmo assim acho que não teria problema alterar :)